db/migrate/20170328110106_fix_expression_in_tenant_quota_report.rb

```diff
@@ -0,0 +1,20 @@
+class FixExpressionInTenantQuotaReport < ActiveRecord::Migration[5.0]
+  class MiqReport < ActiveRecord::Base; end
+
+  OLD_VALUE = 'count: tenants.tenant_quotas'.freeze
+  NEW_VALUE = 'count: Tenant.tenant_quotas'.freeze
+
+  def up
+    MiqReport.where(:db => 'Tenant', :rpt_type => 'Custom').each do |x|
```

Comment:
We don't have many `MiqReport` entries, but you may want to add:

```ruby
where("conditions =~ '%#{OLD_VALUE}%'")
```

I'm good if you don't make that change